.github/workflows/clien-extract-text.yml

```
@@ -0,0 +1,57 @@
+
+jobs:
+  deploy-test-lambda:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write  # GitHub OIDC 사용을 위한 설정
+      contents: read
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)